exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/WebServer.java

```diff
@@ -136,7 +136,8 @@ public void start() throws Exception {
     final boolean portHunt = config.getBoolean(ExecConstants.HTTP_PORT_HUNT);
     final int acceptors = config.getInt(ExecConstants.HTTP_JETTY_SERVER_ACCEPTORS);
     final int selectors = config.getInt(ExecConstants.HTTP_JETTY_SERVER_SELECTORS);
-    final QueuedThreadPool threadPool = new QueuedThreadPool(2, 2, 60000);
+    final int handlers = config.getInt(ExecConstants.HTTP_JETTY_SERVER_HANDLERS);
+    final QueuedThreadPool threadPool = new QueuedThreadPool(2, 2);
```

Comment:
Given another thought to this since we are hard-coding number of threads as well, no need to make idle time configurable. Please disregard the question.

Reply:
@arina-ielchiieva Do you refer to min number of threads? The max number of threads is changed later based on # of handlers, selectors, and connectors.

Reply:
Well, to both but now I understand why we hard-code them. Thanks.

Reply:
Welcome. I also don't see a need to make idle time configurable as Drill WebUI should not be used for hundreds of concurrent queries/requests.